Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BoxViewportEdit.cpp

```diff
@@ -18,6 +18,11 @@
 
 namespace AzToolsFramework
 {
+    namespace
+    {
+        const float MinBoxDimension = 0.001f;
+    } // namespace
+
```

Comment:
This anonymous namespace usage is not compiling for VS 2019 with other recent changes. Please use a fully qualified namespace to avoid symbol redefinition errors.

Reply:
Done.